docs/source/evaluations/alan_turing_institute.md

```diff
@@ -333,3 +333,397 @@
+    - We intend to iterate on the design of the user guide to make it easier to navigate, follow and understand - and separate it entirely from developer docs.
+* - 2.1.6.
+  - 2
+  - We use the Azure platform-level automation tools to run weekly software updates on all virtual machines that make up the TRE.
+    Any update failures are flagged by the automation software.
+* - 2.1.7.
+  - 2
+  - Within each project environment we have a range of shared services.
+    These include shared folders, user services such as GitLab, for collaborating on code, CodiMD, for collaborating on document writing and several database systems.
+* - 2.1.8.
+  - 2
+  - These shared services are only available to users working within the same environment.
+* - 2.1.9.
+  - 2
+  - User-facing software and tools are all open source.
+    We do not allow any software to contact external licensing servers.
+* - 2.1.10.
+  - 2
+  - We provide a wide range of tools and applications for data science, influenced by the needs of users.
+    Our users are typically data scientists working with data directly.
+    This data can only be accessed from inside the TRE, either via a database or a shared folder.
+* - **Capability met?**
+  - **YES**
+  -
+```
+
+## Code Version Control System
+
+```{list-table}
+:header-rows: 1
+:name: tab-turing-dsh-computing-code-vcs
+
+* -
+  - Score
+  - Response
+* - 2.1.11.
+  - 1
+  - - Version control tools are provided to users, including an internal GitLab instance.
+    - Users are encouraged to version control their code and we provide training for those who are unfamiliar with git.
+    ##### Potential Improvements
+    - We do not provide specific tools to aid or encourage reproducibility or creating data analysis pipelines.
+    - We do not support CI pipelines on our GitLab server.
+    - We do not have a method, such as containerisation, to ensure that work done inside the environment can be reproduced outside.
+* - **Capability met?**
+  - **YES**
+  -
+```
+
+## Artefact Management Application
+
+```{list-table}
+:header-rows: 1
+:name: tab-turing-dsh-computing-artefact-management
+
+* -
+  - Score
+  - Response
+* - 2.1.12.
+  - 2
+  - We provide proxied access to external software repositories, currently PyPI and CRAN, using Sonatype Nexus.
+    For our highest sensitivity projects we instead provide a local mirror.
+    In either case, we can support either access to every package in the remote repository or a pre-specified allowed list of approved packages.
+* - 2.1.13.
+  - 2
+  - - We do restrict access to a pre-specified allowed list for higher sensitivity environments.
+      These allowed lists include a minimal set of well-used and useful packages plus their dependencies.
+    - We have guidelines about how to request new packages which involve making a request on our GitHub repository.
+      Requests are then approved or denied by sysadmins based on whether their usefulness outweighs any potential risk to the project.
```

Comment:
Are we mixing up the process for core packages and package proxies here?

It might be clearer just to explain that the allowlists are configurable per project and we provide sensible defaults.

Reply:
Done